src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int RESIZE_MULTIPLIER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] values;
```

Comment:
think should be better rename this variable
```suggestion
    private Node<K, V>[] table;
```